RecoLocalMuon/GEMRecHit/python/me0RecHits_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+me0RecHits = cms.EDProducer("ME0RecHitProducer",
+    recAlgoConfig = cms.PSet(
+
+    ),
```

Comment:
The blank line for PSet parameters is a bit confusing. Is a parameter missing? Please delete the blank space or add an explanatory comment.